### PR TITLE
adds example FIP ophys rig

### LINKED ISSUE #391: fiber photometry example rig

### PR DESCRIPTION
**Pull request by dyf**
closes #391

### CODE REVIEW

File: src/aind_data_schema/device.py [hunk 1]
```diff
@@ -401,6 +403,8 @@ class CameraTarget(Enum):
     BOTTOM = "Bottom"
     EYE = "Eye"
     FACE = "Face"
+    FACE_BOTTOM = "Face side"
```
Comment (saskiad) [inline src/aind_data_schema/device.py:406]: First, this needs to be "Face bottom"
Second, if we have face bottom and face side, what is face?

File: examples/fip_ophys_rig.py [hunk 2]
```diff
@@ -0,0 +1,245 @@
...
+        d.LightEmittingDiode(name="565nm LED", manufacturer=d.Manufacturer.THORLABS, model="M565F3", wavelength=415),
+    ],
+    detectors=[
+        ophr.Detector(
+            name="FLIR CMOS for Green Channel",
+            serial_number="21396991",
+            manufacturer=d.Manufacturer.FLIR,
+            model="BFS-U3-20S40M",
+            notes=(
+                "SpinViewSettings - Binning:4*4, Binning mode: Additive, Crop: 200*200 pixels, Pixel Format: Mono 16"
```
Comment (saskiad) [inline examples/fip_ophys_rig.py:83]: if you're updating ophys rig already, let's add these to the detectors. This is one of the things waiting on my ability to use git.

**Review by saskiad — CHANGES_REQUESTED** [review]
couple of things need to be fixed

File: src/aind_data_schema/device.py (hunk 1 above)
Comment (dyf) [reply src/aind_data_schema/device.py:406]: We could get rid of `FACE`?

File: examples/fip_ophys_rig.py [hunk 3]
```diff
@@ -0,0 +1,253 @@
...
+        ophr.Detector(
+            name="FLIR CMOS for Green Channel",
+            serial_number="21396991",
+            manufacturer=d.Manufacturer.FLIR,
+            model="BFS-U3-20S40M",
+            detector_type="Camera",
+            data_interface="USB",
+            cooling="air",
+            immersion="air",
+            bin_width=4,
```
Comment (saskiad) [inline examples/fip_ophys_rig.py:86]: bin needs a unit. pixels?

File: examples/fip_ophys_rig.py [hunk 4]
```diff
@@ -0,0 +1,253 @@
...
+            bin_height=4,
+            bin_mode="additive",
+            crop_width=200,
```
Comment (saskiad) [inline examples/fip_ophys_rig.py:89]: crop needs a unit too

**Review by saskiad — CHANGES_REQUESTED** [review]
just need units for the new fields

**Review by saskiad — APPROVED** [review]

**Pull request merged**